src/components/SharedComponents/PostList/PostList.tsx

```diff
@@ -97,7 +96,12 @@ const PostList: FC<PostListProps> = (props) => {
       ) : null}
       {withPagination && (
         <Box $width="100%" $mt={[0, "auto"]} $pt={48}>
-          <Pagination pageName={blogsOrWebinars} {...paginationProps} />
+          <OakPagination
+            {...paginationProps}
+            onPageChange={onPageChange}
```

Comment:
tiny thing but `onPageChange` is coming from the paginationProps now so doesn't need to be added separately

Reply:
Refactored ✅